thunks.go

```diff
@@ -181,6 +181,8 @@ func (f *bindingsUnboundField) bindToObject(sb selfBinding, origBindings binding
 	return f.inner.bindToObject(sb, upValues, fieldName)
 }
 
+// PlusSuperUnboundField represents a `field+: ...` that hasn't been bound to a
+// super object.
```

Comment:
`s/a super object/an object/`

It's not bound to any object. "not bound to a super object" suggests that it may be bound to an object in which it appeared.